mlptrain/configurations/configuration_set.py

```diff
@@ -318,42 +336,113 @@ def save_xyz(
         return None
 
     def load_xyz(
-        self, filename: str, charge: int, mult: int, box: Optional[Box] = None
+        self,
+        filename: str,
+        charge: int,
+        mult: int,
+        box: Optional[Box] = None,
+        load_energies: bool = False,
+        load_forces: bool = False,
     ) -> None:
         """
-        Load configurations from a .xyz file. Will not load any energies or
-        forces
+        Load configurations from a .xyz file with optional box, energies and forces if specified.
 
         -----------------------------------------------------------------------
         Arguments:
-            filename:
+            filename: name of the input .xyz file
+
+            charge: total charge on all configurations in the set
 
-            charge: Total charge on the configuration
+            mult: total spin multiplicity on all configurations in the set
```

Comment:
It is worth mentioning that this function assumes that all data have the same charge and multiplicity.

Reply:
Will add this!